src/main/analytics/user-handlers.ts

```diff
@@ -2,15 +2,43 @@ import { ipcMain } from 'electron';
 
 import eventBus from '../event-bus';
 import { userLogout, userSignin, userSigninFailed } from './service';
+import { RemoteSyncManager } from '../remote-sync/RemoteSyncManager';
+import { DriveFilesDB } from '../database/driveFilesDB';
+import { DriveFoldersDB } from '../database/driveFoldersDB';
+import Logger from 'electron-log';
+import { clearRemoteSyncStore } from '../remote-sync/helpers';
+import { reportError } from '../bug-report/service';
 
 eventBus.on('USER_LOGGED_IN', () => {
-	userSignin();
+  userSignin();
+  const remoteSyncManager = new RemoteSyncManager(
```

Comment:
We should not initialize the local db on the analytics "module". The structure is not perfect, let's try to keep it as clean as possible. `USER_LOGGED_IN` can be listened to as long as you are on the main process.

Reply:
just added this here since I don't know yet how are we going to use this and when, but we can move to your way

Reply:
Moved this to own remote sync handlers file